src/screen/splash/mod.rs

```diff
@@ -2,26 +2,28 @@
 
 use bevy::{asset::embedded_asset, prelude::*};
 
-use crate::util::prelude::*;
-
 use super::Screen;
+use crate::util::prelude::*;
 
 pub(super) fn plugin(app: &mut App) {
     // Add splash image
     embedded_asset!(app, "splash.png");
-    app.add_systems(OnEnter(Screen::Splash), spawn_splash)
-        // Make the screen-flicker bug less jarring on Windows (see `core/deflicker.rs`).
-        .insert_resource(ClearColor(SPLASH_BACKGROUND_COLOR));
+    app.add_systems(OnEnter(Screen::Splash), spawn_splash);
+    // Make the screen-flicker bug less jarring on Windows
+    // (see `core/deflicker.rs`).
```

Comment:
Hm this isn't part of your PR, but this comment is outdated. It could be changed to just "Make the screen-flicker bug less jarring." or no comment at all.

Reply:
Feel free to skip this, can be updated in another PR.

Reply:
Good catch
[`12c48d2` (#78)](https://github.com/TheBevyFlock/bevy-template/pull/78/commits/12c48d22b922bd0dd66497b1c91f265add64871c)